lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -145,12 +145,10 @@ sub error_403 : Private
     $c->stash->{template} = 'main/403.tt';
 }
 
-sub error_404 : Private
-{
+sub error_404 : Private {
     my ($self, $c) = @_;
 
     $c->response->status(404);
-    $c->stash->{template} = 'main/404.tt';
```

Comment:
Unrelated to the ticket but related to the 404 page...I forgot to remove the old root/main/404.tt so I'm doing that in https://github.com/mwiencek/musicbrainz-server/commit/0fd5a2885cb1242d45ca58cf8aaee395a8731b8d